packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -62,51 +67,101 @@ const OrderRowComponent = observer(({ data: order, style }) => {
 
     const offer_amount = `${amount_display} ${account_currency}`;
     const transaction_amount = `${price_display} ${local_currency}`;
+    const is_buy_order_type_for_user = (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad);
+    const order_type = is_buy_order_type_for_user ? localize('Buy') : localize('Sell');
+
+    const Title = () => {
+        const send_amount = is_buy_order_type_for_user ? transaction_amount : offer_amount;
+        return (
+            <React.Fragment>
+                <Text size='sm' line_height='xxs' weight='bold' as='p'>
+                    {order_type} {send_amount}
+                </Text>
+                <Text color='less-prominent' as='p' line_height='xxs' size='xxs' align='left'>
+                    {order_purchase_datetime}
+                </Text>
+            </React.Fragment>
+        );
+    };
```

Comment:
In general, try not to create components within components. React will keep creating a new `Title` component on the re-render of this `OrderRowComponent so it'll be unmounted and re-mounted on the DOM on each render. If you'd define the `Title` component outside the `OrderRowComponent` it would not unmount and remount.

Reply:
@aaron-binary 
Thanks for your comment 🌹 
Code updated and now Title defined outside of OrderRowComponent